internal/web3ext/web3ext.go

```diff
@@ -772,6 +772,11 @@ web3._extend({
 			call: 'txpool_contentFrom',
 			params: 1,
 		}),
+		new web3._extend.Method({
+			name: 'remove',
+			call: 'txpool_remove',
```

Comment:
`txpool_removeTransactionByHash` might be more self-explanatory.

Reply:
done